bitnami/drupal/templates/deployment.yaml

```diff
@@ -72,7 +72,7 @@ spec:
             - name: BITNAMI_DEBUG
               value: {{ ternary "true" "false" .Values.image.debug | quote }}
             - name: ALLOW_EMPTY_PASSWORD
-              value: {{ ternary "yes" "no" .Values. allowEmptyPassword | quote }}
+              value: {{ .Values.allowEmptyPassword | quote }}
```

Comment:
The problem was the space between "Values" and "allowedEmptyPassword". Why do you remove the ternary?

Reply:
`.Values.allowEmptyPassword` is not a boolean, it's a `yes/no` string so it doesn't work with the ternary operator. I'm doing what is done in other standardized charts.